scripts/postman_test_automation.sh

```diff
@@ -19,16 +19,23 @@ get_api_keys() {
     # [<connector_name>]
     # api_key = "HeadKey of <connector_name>"
 
-    API_KEY=$(echo "${result}" | awk -F ' = ' '$1 == "api_key" { print $2 }')
-    KEY1=$(echo "${result}" | awk -F ' = ' '$1 == "key1" { print $2 }')
-    API_SECRET=$(echo "${result}" | awk -F ' = ' '$1 == "api_secret" { print $2 }')
-
+    # Keys are exported as environment variables since some API Keys for connectors such as ACI
+    # which is Base64 based and requires "Bearer" to be prefixed such as "Bearer Skst45645gey5r#&$==".
```

Comment:
```suggestion
    # Keys are set as variables since some API Keys for connectors such as ACI
    # are Base64 encoded and require "Bearer" to be prefixed such as "Bearer Skst45645gey5r#&$==".
```

Make changes as you see fit.